internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -127,6 +161,24 @@ func (h *HighPerformanceHooks) PreStop(ctx context.Context, c *oci.Container, s
 
 	// no need to reverse the cgroup CPU CFS quota setting as the pod cgroup will be deleted anyway
 
+	// restore the c-state configuration for the container CPUs
+	configureCState, _ := shouldCStatesBeConfigured(s.Annotations())
+	if configureCState {
```

Comment:
can we note why we don't need to restore the state when it was disabled?

Reply:
I added a comment. We only restore the c-state when the annotation is present - without the annotation we don't modify the c-state when the container is started or stopped.